CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.10.x
 
 Breaking changes:
+- [#1535](https://github.com/rails-api/active_model_serializers/pull/1535) Move the adapter and adapter folder to
```

Comment:
Should the "ApiObjects" also be re-add or should I simply change the references to match with the `ActiveModelSerializers` namespace?